Automattic/jetpack
# JITMS: adding new JITM styles

Adding 2 new JITM styles and cleaning up the original JITM

To do:
- [x] clean up woo commerce JITM (visually)
- [ ] clean up original JITM styles. Update outdated css
- [x] add styles for new JITM banners (calypso styled)

**Current JITM style example:**
<img width="776" alt="old-jitm-style" src="https://cloud.githubusercontent.com/assets/214813/26079357/7ec7639e-3990-11e7-8852-ea2998db6ac6.png">


**New JITM style examples:**
<img width="1085" alt="screen shot 2017-05-15 at 4 59 26 pm" src="https://cloud.githubusercontent.com/assets/214813/26079323/61131032-3990-11e7-8651-e43519e37c04.png">

* Can change border color if desired.
* Can have title + description or just a title
* Can have gray or primary blue CTA button
* JITMs will dynamically resize/realign based on the content. In order words, If you only have a title in the JITM, it'll be vertically centered, and when you add a description, it'll automatically ensure both are vertically centered. ;-)
*Work well on mobile

Important to note that the CTA buttons have a max width in order to be easier to predict and control how they'll be displayed on various screen sizes—which can bump the button text to two lines if its long. We should try and re-word some of the JITMS so the button is one word and most of the verbs are in the description.


**TO TEST:**

1. Get this branch synced up to your sandbox.
2. Visit `yousite.com/wp-admin/update-core.php` where the sample JITMs sit ** Ensure VaultPress is deactivated to see these particular JITMs
3. Sample woo JITM is hanging out at `yoursite.com/wp-admin/edit.php?post_type=shop_order` when woo commerce is installed.

** If you've dismissed JITMs in the past and no longer see them, you'll need to reset your JITM options. Do so by SSH'ing into your sandbox and clearing them with `cd public_html` and then `wp jetpack options delete hide_jitm`
